src/EPiServer.SocialAlloy.Web/Social/Repositories/ActivityStreams/CommunityFeedRepository.cs

```diff
@@ -77,11 +74,9 @@ public IEnumerable<CommunityFeedItemViewModel> Get(CommunityFeedFilter filter)
             {
                 throw new SocialRepositoryException("Episerver Social failed to process the application request.", ex);
             }
-
-            return AdaptFeedItems(feedItems);
         }
 
-        private IEnumerable<CommunityFeedItemViewModel> AdaptFeedItems(List<Composite<FeedItem, CommunityActivity>> feedItems)
+        private IEnumerable<CommunityFeedItemViewModel> AdaptFeedItems(List<FeedItem<CommunityActivity>> feedItems)
```

Comment:
The adapter used in this method is one of the key areas where changes will need to be made regarding the handling of activity data on feed items.

Previously, SocialAlloy leveraged a base CommunityActivity class that was extended by a CommentActivity and RatingActivity. It then used a visitor to handle the different types uniquely. It may be easiest to just settle on a common format for simply a CommunityActivity. That will streamline things and eliminate the adaptation complexity of trying to work with multiple different types at once.